src/main/java/org/traccar/protocol/TelemaxProtocolDecoder.java

```diff
@@ -97,7 +97,7 @@ protected Object decode(
                 time = new SimpleDateFormat("yyMMddHHmmss").parse(readValue(sentence, index, 12));
                 position.set(Position.KEY_STATUS, readValue(sentence, index, 8));
             } else {
-                time = new Date(time.getTime() + interval * 1000);
+                time = new Date(time.getTime() + interval * 1000L);
```

Comment:
This is incorrect. `interval` is an int, so if we don't multiply by a long explicitly, it will overflow.

Reply:
ok, so you agree with this change, right? Because it suggest to multiply by a long.